pkgs/development/libraries/gettext/default.nix

```diff
@@ -47,6 +47,10 @@ stdenv.mkDerivation rec {
     sed -i -e "s/\(libgettextsrc_la_LDFLAGS = \)/\\1..\/gnulib-lib\/libxml_rpl.la /" gettext-tools/src/Makefile.in
   '' + lib.optionalString stdenv.hostPlatform.isMinGW ''
     sed -i "s/@GNULIB_CLOSE@/1/" */*/unistd.in.h
+  ''
+    # PR: https://github.com/NixOS/nixpkgs/issues/348658
```

Comment:
This isn't a lot of explanation for a very confusing fix…

Do we know why it's being redeclared?

Reply:
I spent like 15 minutes looking into it and got completely stumped, this is the best solution I was able to find lol. The macros for things like having the config, libc, and getcwd exists so theoretically it should just work.

Reply:
```suggestion
    # ISSUE: https://github.com/NixOS/nixpkgs/issues/348658
```

Reply:
Typo lol, I'm so used to writing PR with Nixpkgs LLVM workspace.